src/snowflake/snowpark/_internal/tcm/proto/ast.proto

```diff
@@ -2486,4 +2486,4 @@ message MapType {
 // type.ir:56
 message TyVar {
   string id = 1;
-}
\ No newline at end of file
+}
```

Comment:
this is a generated file. cool if ir-dsl-c is updated so that this diff is a result of running ast-make.sh.